README.md

```diff
@@ -6,5 +6,11 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails exists as a backend. It is used to facilitate interaction between two people (for example, playing a game online). It can also store the state of a game if the user wants to come back to it later without having to start over. It also stores registration and login information.
+
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
The front-end part of your diagram is mostly right, but the back-end part is missing a lot of details. For instance, how does the request get parsed? How does data get added to (/retrieved from) our records?